packages/abi-gen/templates/Python/partials/method_class.handlebars

```diff
@@ -26,7 +26,7 @@ class {{toPythonClassname this.languageSpecificName}}Method(ContractMethod):
         return ({{> params }})
 
     {{/if}}
-    def call(self, {{#if inputs}}{{> typed_params inputs=inputs}}, {{/if}}tx_params: Optional[TxParams] = None) -> {{> call_return_type outputs=outputs type='call'~}}:
+    def call(self, {{#if inputs}}{{> typed_params inputs=inputs}}, {{/if}}tx_params: Optional[TxParams] = None, state_override: Optional[StateOverride] = None) -> {{> call_return_type outputs=outputs type='call'~}}:
```

Comment:
Ah cool, didn't know you were also adding it to the Python wrappers. Cool.

Could we potentially just pass a `custom_params` param in, which is an object with a single key to start (`state_override`)? Just thinking that we might as well future-proof this so if we need to pass in additional optional params to `call` in the future, we don't need to introduce a breaking change. thoughts?

Reply:
So it turns out that the Python web3 library does not allow to pass in the optional third parameter to call [0], therefore I removed `StateOverride` from the python templates altogether.

We could probably add it, but that would require calling `self.web3.manager.request_blocking` directly [1].

Should I add support in Python for this?

[0] - https://web3py.readthedocs.io/en/stable/web3.eth.html
[1] - https://github.com/ethereum/web3.py/blob/master/web3/eth.py#L428